mypy/build.py

```diff
@@ -499,7 +499,7 @@ def __init__(self, data_dir: str,
                  search_paths: SearchPaths,
                  ignore_prefix: str,
                  source_set: BuildSourceSet,
-                 reports: Optional['Reports'],
+                 reports: 'Optional[Reports]',
```

Comment:
this one is subtle, `pyflakes` knows how to look at string-annotations but hasn't yet figured out string-generics, there's an open issue for this however (will need to handle `Literal['foo']` differently, and a few other things (why it wasn't tackled quite yet)) https://github.com/PyCQA/pyflakes/issues/447